Tools/TestWebKitAPI/Tests/ios/AutocorrectionTestsIOS.mm

```diff
@@ -125,6 +125,8 @@ static void checkCGRectIsEqualToCGRectWithLogging(CGRect expected, CGRect observ
 
 TEST(AutocorrectionTests, AutocorrectionContextDoesNotIncludeNewlineInTextField)
 {
+    if ([UIKeyboard usesInputSystemUI])
```

Comment:
you might need to import the SPI header, or forward declare this method again to fix the build errors

Reply:
Thanks! I forgot to declare it in TestWebKitAPI's UIKitSPI header